src/app/pages/instruction-viewer/instruction-viewer.component.sass

```diff
@@ -34,12 +34,17 @@
 #partlist-content
   width: 100%
   display: flex
-  background-color: #f2f2f2
+  background-color: transparent
```

Comment:
yeaaah this is self explanatory but might need different handling

if you want white background in those boxes, change clear color in the rendering part